docs/components/console/manage-clusters/delete-cluster.md

```diff
@@ -8,10 +8,6 @@ description: "Follow these step-by-step instructions to remove your cluster perm
 This action cannot be undone.
 :::
 
-A cluster can be deleted at any time. To delete your cluster, follow the steps below:
+A cluster can be deleted at any time. To delete your cluster, navigate to the **Clusters** tab in the top navigation and click **Delete** to the far right of the cluster name.
 
-1. Open the cluster details by clicking on the cluster name.
-2. Select the three vertical dots next to the cluster name near the top of the page to open the cluster's menu.
-3. Click **Delete**.
-
-![cluster-delete](./img/cluster-delete.png)
+<!--- Screenshot outdated -->
```

Comment:
I didn't see this in https://github.com/camunda/camunda-docs/pull/2757/files so this should be updated in the scope of this PR.

Reply:
@jprcoliveira can you assist here?